lobster/tools/codebeamer/codebeamer.py

```diff
@@ -535,18 +537,14 @@ def main():
 
     elif options.import_query:
         try:
-            if isinstance(options.import_query, int):
-                query = int(options.import_query)
-                if query < 1:
-                    ap.error("query_string must be a positive integer")
-            elif isinstance(options.import_query, str):
-                if options.import_query.startswith("-"):
-                    ap.error("query_string must be a positive integer"
-                                     " or valid string")
+            if isinstance(options.import_query, str):
+                if (options.import_query.startswith("-") and
+                    options.import_query[1].isdigit()):
+                    ap.error("import-query must be a positive integer")
+                elif options.import_query.startswith("-"):
+                    ap.error("import-query must be a valid cbQL query")
                 elif options.import_query.isdigit():
                     query = int(options.import_query)
-                    if query < 1:
-                        ap.error("query_string must be a positive integer")
                 else:
```

Comment:
I think this statement is not needed by default it is string. I would suggest please try to test without this statement